packages/html-reporter/src/testResultView.tsx

```diff
@@ -135,7 +136,10 @@ export const TestResultView: React.FC<{
       </div>)}
     </AutoChip>}
 
-    {!!otherAttachments.size && <AutoChip header='Attachments'>
+    {!!(otherAttachments.size + htmls.length) && <AutoChip header='Attachments'>
+      {[...htmls].map((a, i) => (
```

Comment:
Not sure if it bothers us, but attachments of different types are out of order now. I.e. if I have text, html, text, html sequence they end up grouped together.

Reply:
that was already the case for screenshots, videos and traces. I guess one could think of them differently because they're "special" attachments, but the same argument could be made for HTML attachments really.

🤷